### PR TITLE
Refactor: Migrate `StakingHierarchy` to Chakra

### PR DESCRIPTION
**Pull request by TylerAPfledderer**
## Description

Migrate the `StakingHierarchy` component to Chakra.

Also includes updating the glyph icons to have `currentColor` as their fill values.

## Related Issue
#8632

### CODE REVIEW

**Review by pettinarip — CHANGES_REQUESTED** [review]
Looking good @TylerAPfledderer 

Just a few things I see comparing it with the prod version:

- small gap between sections
image

- eth icons are too small on small resolutions
image

This is in prod:
image

**Comment by TylerAPfledderer** [comment]
@pettinarip yikes! We have browser inconsistencies here. I definitely see this in Chrome, but the styling works as expected in Firefox.

**Review by pettinarip — COMMENTED** [review]
Thanks for the fixes @TylerAPfledderer.

I wouldn't say this is a blocker but I still see a small gap between the line and the eth logos. Probably it is related to the `VStack` but I see you set it as `gap={0}` in the `md` breakpoint, so I'm not sure what is going on.

image

File: src/components/Staking/StakingHierarchy.tsx [hunk 1]
```diff
@@ -15,258 +14,259 @@ import CexGlyph from "../../assets/staking/staking-glyph-centralized.svg"
...
+          left: calc.subtract("50%", "2px"),
+          top: "50px",
+          zIndex: 1,
+        }}
+      />
+    </Show>
+  )
+}
 
-  .gold {
```
[the remark this answers is not in the document]
Comment (TylerAPfledderer) [reply src/components/Staking/StakingHierarchy.tsx:194]: I will add it back in and add a comment above it stating that markdown files use this classname

**Review by Fqc301033 — APPROVED** [review]

**Comment by TylerAPfledderer** [comment]
> Thanks for the fixes @TylerAPfledderer.
> 
> I wouldn't say this is a blocker but I still see a small gap between the line and the eth logos. Probably it is related to the `VStack` but I see you set it as `gap={0}` in the `md` breakpoint, so I'm not sure what is going on.
> 
> image

With `VStack` I am supposed to use the `spacing` prop and not the `gap` prop (renders the "owl" selector to create margins). So even with `gap` margin still existed. 🤦🏼 

For context and example, if `spacing={2}`, then the following selector and property gets rendered for every sibling after the first:
```css
.css-tnhm38 > :not(style) ~ :not(style) {
  margin-top: 0.5rem;
}
```

`gap` would only really work if you were using the `Flex` component, say, or any component that doesn't do the above selector rendering under the hood.

**Comment by TylerAPfledderer** [comment]
@pettinarip should be good to go now! 😄

**Comment by pettinarip** [comment]
Oh got it. makes sense now :)

**Review by pettinarip — APPROVED** [review]
lgtm!

**Pull request merged**